Multisig/UI/Settings/OwnerKeyManagement/LedgerOwnerKey/LedgerPendingConfirmationViewController.swift

```diff
@@ -55,5 +55,24 @@ class LedgerPendingConfirmationViewController: UIViewController {
     private func close() {
         dismiss(animated: true)
     }
-}
 
+    var highlightedLedgerHash: NSAttributedString {
+        let value = ledgerHash!
+        let style = GNOTextStyle.tertiary
+        let attributedString = NSMutableAttributedString(
```

Comment:
How about create new text style for this ?

Reply:
dunno, if that's used everywhere or not